### PR TITLE
[APP-1979] Fix react native bugs

### PR DESCRIPTION
**Pull request by KseniaBuntova**
## Technical description

- [x] Feature description...

## Tested on

-
- 

## Steps to reproduce

1. 
2. 

## Screenshots (optional):

No screenshots.

*This Pull Request template has been written and generated by Monk JS repository.*

### CODE REVIEW

File: packages/camera/src/components/Sights/index.js [hunk 1]
```diff
@@ -19,7 +19,6 @@ const styles = StyleSheet.create({
     paddingVertical: 6,
     alignSelf: 'center',
     color: 'white',
-    fontFamily: 'monospace',
```
Comment (artgoncharov) [inline packages/camera/src/components/Sights/index.js:22]: check if web

File: packages/camera/src/components/AddDamageModal/assets/RotateRight.js [hunk 2]
```diff
@@ -11,7 +11,6 @@ export default function RotateRight({ width, height }) {
       xmlns="http://www.w3.org/2000/svg"
       viewBox="0 0 16 37"
       preserveAspectRatio="xMidYMid slice"
-      pointerEvents="box-none"
```
Comment (artgoncharov) [inline packages/camera/src/components/AddDamageModal/assets/RotateRight.js:14]: check if web

File: packages/camera/src/components/Capture/capture.js [hunk 3]
```diff
@@ -45,6 +45,9 @@ const ADD_DAMAGE_HELP_EVENT_KEY = 'ADD_DAMAGE_HELP';
 const styles = StyleSheet.create({
   container: {
     flex: 1,
+    height: '100%',
```
Comment (artgoncharov) [inline packages/camera/src/components/Capture/capture.js]: remove

File: packages/camera/src/components/UploadCenter/index.js [hunk 4]
```diff
@@ -162,6 +162,35 @@ export default function UploadCenter({
...
+        ) : null}
+        <Button
+          colors={colors}
+          color={colors.actions.secondary}
+          onPress={(e) => {
+            log(['[Click] Skip retaking photo']);
+            onSkipRetake(statEventsData);
+            onComplianceCheckFinish(e);
+          }}
+          disabled={isSubmitting || hasAllRejected
```
Comment (artgoncharov) [inline packages/camera/src/components/UploadCenter/index.js]: revert

File: packages/camera/src/components/AddDamageModal/SVGComponentMapper.native.js [hunk 5]
```diff
@@ -0,0 +1,128 @@
...
+
+export default function SVGComponentMapper({ element, togglePart, isPartSelected, groupName }) {
+  let names = [];
+  for (let i = 0; i < element.attributes.length; i++) {
+    names.push(element.attributes[i].name);
+  }
+
+  const attributes = useMemo(() => names.reduce((prev, attr) => ({
+      ...prev,
+      [jsxSpecialAttributes[attr] ?? attr]: element.getAttribute(attr),
```
Comment (artgoncharov) [inline packages/camera/src/components/AddDamageModal/SVGComponentMapper.native.js]: getAttribute is not available in native

File: packages/camera/src/components/Capture/hooks.js [hunk 6]
```diff
@@ -190,16 +189,14 @@ export function useStartUploadAsync({
 }) {
   const [queue, setQueue] = useState([]);
   const { errorHandler } = useMonitoring();
-  let isRunning = false;
-
-  const addElement = useCallback((element) => setQueue((prevState) => [...prevState, element]), []);
```
Comment (artgoncharov) [inline packages/camera/src/components/Capture/hooks.js]: revert

File: packages/camera/src/components/AddDamageOverlay/index.native.js [hunk 7]
```diff
@@ -1,24 +1,84 @@
-import React from 'react';
+import React, { useMemo } from 'react';
 import PropTypes from 'prop-types';
-import { SvgXml } from 'react-native-svg';
+import { StyleSheet, useWindowDimensions, View } from 'react-native';
+
+const styles = StyleSheet.create({
```
Comment (artgoncharov) [inline packages/camera/src/components/AddDamageOverlay/index.native.js]: remove

**Review by artgoncharov — APPROVED** [review]

**Review by knoskov — APPROVED** [review]

**Pull request merged**